dbclient/test/HiveClientTest.py

```diff
@@ -41,4 +41,36 @@ def mock_submit_cmd(cid, ec_id, repair_cmd):
             hiveClient.repair_legacy_tables(fix_table_log='test_repair_tables.log')
             output = fake_out.getvalue().split("\n")
             self.assertEqual(output[0], 'Table failed repair: default.test_legacy2')
-            self.assertEqual(output[1], '1 tables failed to repair. See errors in failed_repair_tables.log')
\ No newline at end of file
+            self.assertEqual(output[1], '1 tables failed to repair. See errors in failed_repair_tables.log')
```

Comment:
Minor correction, `1 table(s)` should be singular.

Reply:
✔️